packages/@webex/plugin-meetings/src/common/browser-detection.ts

```diff
@@ -9,7 +11,7 @@ const mockDetectionObject = {
   /* eslint-enable global-require */
   getBrowserName: () => '',
   getBrowserVersion: () => '',
-  isBrowser: () => false
+  isBrowser: () => false,
```

Comment:
another example of formatting change - I think it's all because of the rename from js to ts and ts files have different formatting rules. @arun3528 this is going to be a problem now even if we try to change the formatting rules for ts files to match js files, because we already have many new ts files on the sdk_v3 branch with the current formatting rules for ts. To be honest this makes me think that we should not do this js to ts conversation now at all and ony do it after sdk_v3 branch becomes the "main" branch or gets merged into master.

Reply:
Most of the formatting are reverted and we have very minor formatting due to added types and couple of typo fixes as well and along with that 2-3 files may have some jsDoc formatting. Please go through the changes and let me know if this much will cause trouble or is it fine.